CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## [[7.3.0]](https://github.com/iExecBlockchainComputing/iexec-sms/releases/tag/v7.3.0) 2023-01-18
+
+* Add health endpoint to allow health checks mechanism enablement.
```

Comment:
Please remember to update this line if you accept [this comment](https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/pull/70#discussion_r1073228673).